ArgoTests/JSON/types.json

```diff
@@ -11,6 +11,7 @@
     "hello",
     "world"
   ],
+  "string_array_opt": null,
```

Comment:
We should actually try decoding this into something in the tests too. As it is, we never touch this value.